app/services/bulk_api_import/fhir_patient_importer.rb

```diff
@@ -0,0 +1,91 @@
+class BulkApiImport::FhirPatientImporter
+  include BulkApiImport::FhirImportable
```

Comment:
Why not use inheritance?

Reply:
A few reasons behind this:
* I intended to model `FhirImportable` as behaviour, and it does not encapsulate any state.
* I don't see us instantiating and using this as an independent object ever.
* I don't see much benefit to having an inheritance hierarchy here—it's as reusable as a mixin, but comes with some more baggage like the lack of composability, IMO

Open to changing my mind for this if you favour inheritance here